app/src/main/java/com/kodeco/android/countryinfo/repository/CountryRepository.kt

```diff
@@ -4,9 +4,16 @@ import com.kodeco.android.countryinfo.model.Country
 import kotlinx.coroutines.flow.Flow
 
 interface CountryRepository {
-    val countries: Flow<List<Country>>
+    // Remote datasource
+    fun fetchCountries(): Flow<List<Country>>
 
-    suspend fun fetchCountries()
+    // Local datasource
+    fun getCountries(): Flow<List<Country>>
```

Comment:
Note, for these two, you wouldn't normally want to expose different methods for local or remote data sources from the repository to the callsite.  The repository should expose one method and internally it can reconcile local and remote data.  So, this could be condensed into:

```kotlin
fun getCountries(): Flow<List<Country>>